components/prism-sas.js

```diff
@@ -53,7 +53,7 @@
 
 	Prism.languages.sas = {
 		'datalines': {
-			pattern: /^(\s*)(?:(?:data)?lines|cards);[\s\S]+?^;/im,
+			pattern: /^(\s*)(?:(?:data)?lines|cards);[\s\S]+?^(\s*);/im,
```

Comment:
1. Please don't use capturing groups.
2. You don't need the group.

```suggestion
			pattern: /^(\s*)(?:(?:data)?lines|cards);[\s\S]+?^\s*;/im,
```

Reply:
Fixed in latest commit.